dev/com.ibm.ws.security.fat.common/test/com/ibm/ws/security/fat/common/apps/formlogin/FormLoginServletTest.java

```diff
@@ -91,7 +91,11 @@ public void test_doGet() {
                     will(returnValue(new Cookie[] { cookie }));
                     one(cookie).getName();
                     one(cookie).getValue();
-                    one(writer).write(with(any(String.class)));
+                    allowing(writer).write(with(any(String.class)));
+                    one(request).getParameter("logout");
+                    one(request).logout();
+                    allowing(writer).write(with(any(String.class)));
+                    allowing(writer).write(with(any(String.class)));
```

Comment:
Putting `allowing(writer).write(with(any(String.class)));` once is sufficient; the `allowing()` method accounts for 0..N calls to the specified method anywhere, so having that line more than once is redundant.